tools/CMakeLists.txt

```diff
@@ -1,7 +1,18 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
-add_subdirectory(oeedger8r)
+if (USE_CPP_EDGER8R)
+  check_submodule_not_empty(oeedger8r-cpp)
+  set(USING_OEEDGER8R_CPP_AS_SUBMODULE on)
```

Comment:
nit: `s/USING/USE/` for consistency with other options

Reply:
Fixed.